test/setup_test_db.sh

```diff
@@ -4,7 +4,17 @@
 # copied into the test folder, then deleted after.
 cp sql/V*.sql test/sql/
 
-docker run --rm -v $(pwd)/test:/flyway/conf -v $(pwd)/sql:/flyway/sql --network=host boxfuse/flyway clean && \
-docker run --rm -v $(pwd)/test:/flyway/conf -v $(pwd)/test/sql:/flyway/sql --network=host boxfuse/flyway migrate
+PG_DB_CONTAINER_NAME="test"  # Also used in package.json to stop the container after the test
+
+# stop and remove any previously-running test containers
+docker stop "${PG_DB_CONTAINER_NAME}"
+docker rm "${PG_DB_CONTAINER_NAME}"
+
+# set up the Postgres database
+docker run --name "${PG_DB_CONTAINER_NAME}" -e POSTGRES_USER=test -e POSTGRES_PASSWORD=test -e POSTGRES_DB=test_report_db -d -p 5436:5432 postgres:10
+
+#perform cleaning and database migration
+docker run --rm -v $(pwd)/test:/flyway/conf -v $(pwd)/sql:/flyway/sql --network=host flyway/flyway clean && \
```

Comment:
I found that the version of Docker Flyway that I'm running needed the `-cleanDisabled=false` argument in order to run `clean`. If your version doesn't require it but permits it as an option, would you mind adding that line to this script?